redbadger/website-honestly
# Events banner set from Prismic

### Motivation

On the events page we have a static image with a link. We currently have to manually update this and redeploy.

This PR supports a new custom type in Prismic called events-banner that allows setting the link, altText and images for the banner.

### Test plan

- Take a look at the new custom type _Event Banner_ in Prismic staging along with the single instance of it.
- Go to the [events page](https://www-staging.red-badger.com/e8b4adf/events) and observe that the events banner is present with correct link, alttext and images
- Try changing the content of the Events Banner using staging Prismic

### Pre-merge checklist
